test/Microsoft.ML.Tests/Scenarios/Api/Estimators/Evaluation.cs

```diff
@@ -25,8 +25,8 @@ public void Evaluation()
             // Pipeline.
             var pipeline = ml.Data.CreateTextLoader(TestDatasets.Sentiment.GetLoaderColumns(), hasHeader: true)
                 .Append(ml.Transforms.Text.FeaturizeText("Features", "SentimentText"))
-                .Append(ml.BinaryClassification.Trainers.StochasticDualCoordinateAscent(
-                    new SdcaBinaryTrainer.Options { NumThreads = 1 }));
+                .Append(ml.BinaryClassification.Trainers.StochasticDualCoordinateAscentCalibrated(
```

Comment:
>StochasticDualCoordinateAscentCalibrated [](start = 57, length = 40)

Would it still work if it be SDCA uncalibrated?

Reply:
It may not work. The following `Evaluate` is calibrated model specific. Uncalibrated case should use `EvaluateNonCalibrated`.

---
In reply to: [256211883](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256211883) [](ancestors = 256211883)

Reply:
So maybe we should call this one SDCA and other learner SDCAUncalibrated?

---
In reply to: [256223247](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256223247) [](ancestors = 256223247,256211883)

Reply:
I honestly prefer calling it SDCALogisticRegression...

---
In reply to: [256258319](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256258319) [](ancestors = 256258319,256223247,256211883)

Reply:
We use `NonCalibrated` to all non-logistic-regression SDCA names.

---
In reply to: [256262110](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256262110) [](ancestors = 256262110,256258319,256223247,256211883)